plutus-scripts/Spec/WriteScriptFiles.hs

```diff
@@ -24,3 +26,5 @@ writeScriptFiles = liftIO $ do
   PS_1_1.writeTimeRangePolicyScriptV3
   PS_1_1.writeWitnessRedeemerPolicyScriptV3
   PS_1_0.writeIntegerToByteStringPolicyScriptV2
+  PS_1_1.writeBitwisePolicyScriptsV3
```

Comment:
I've only generated scripts for PlutusV3 and version 1.1.0.  We may need to do it for V1/V2 as well (and 1.0.0?) if we later allow the new functions to be used with all ledger language versions.